shiny/experimental/ui/_card.py

```diff
@@ -190,6 +190,7 @@ def card_image(
     container: ImgContainer = card_body,
     **kwargs: TagAttrValue,
 ) -> Tagifiable:
+    # TODO-future: Must be updated to match rstudio/bslib#1076 before moving from exp.
```

Comment:
I don't know if this removes the documentation for the method. Is it ok if we move to just after the docstring?

Reply:
It doesn't, I was following your example actually

https://github.com/posit-dev/py-shiny/blob/6611277da634503e82adcdee3aa1dd67d7bf0a87/shiny/ui/_sidebar.py#L459-L467